From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 01 Dec 2015 02:21:58 +0000
Subject: af_unix: Avoid ABI changes
Forwarded: not-needed

struct unix_sock is only allocated in af_unix so it's safe to add new
members at the end.  Hide peer_wake from genksyms.

---
--- a/include/net/af_unix.h
+++ b/include/net/af_unix.h
@@ -63,7 +63,9 @@ struct unix_sock {
 #define UNIX_GC_CANDIDATE	0
 #define UNIX_GC_MAYBE_CYCLE	1
 	struct socket_wq	peer_wq;
+#ifndef __GENKSYSMS__
 	wait_queue_t		peer_wake;
+#endif
 };
 
 static inline struct unix_sock *unix_sk(struct sock *sk)
